index.html

```diff
@@ -4,13 +4,7 @@
 <head>
   <link rel="stylesheet" type="text/css" href="sty3.css">

```

Comment:
still file name is sty3. We should be naming it as style.css.

Reply:
ok.. i will be changing